docs/user/faq.rst

```diff
@@ -959,12 +959,41 @@ the tutorial in the docs provides an excellent introduction to
 
 (See also: `Testing <http://falcon.readthedocs.io/en/stable/api/testing.html>`_)
 
-How to set cookies in simulate request for testing?
----------------------------------------------------
+How can I set cookies when simulating requests?
+-----------------------------------------------
 
-This can be done by setting ``headers={'Cookie': 'xxx=yyy'}`` in
+The easiest way is to simply pass the ``cookies`` parameter into
 ``simulate_request``. Here is an example:
 
+.. code:: python
+
+    import falcon
+    import falcon.testing
+    import pytest
+
+    class TastyCookies:
+
+        def on_get(self, req, resp):
+            resp.media = {'cookies': req.cookies}
+
+
+    @pytest.fixture
+    def client():
+        app = falcon.App()
+        app.add_route('/cookies', TastyCookies())
+
+        return falcon.testing.TestClient(app)
+
+
+    def test_cookies(client):
+        resp = client.simulate_get('/cookies', cookies={'cookie': 'cookie value'})
+
+        assert resp.json == {'cookies': {'cookie': 'cookie value'}}
+
+
+Alternatively, you can set the Cookie header directly, as showing in the
+following example.
```

Comment:
Not a hard requirement, but maybe we don't need to duplicate the whole file again, and just illustrate the difference in `test_cookies(client)`.

Reply:
Perhaps something like:

> Alternatively, you can set the Cookie header directly as demonstrated in this version of `test_cookies()`:

...and then only show the altered method in the code snippet?